giskard_vision/image_classification/models/wrappers.py

```diff
@@ -0,0 +1,57 @@
+import numpy as np
+
+from giskard_vision.image_classification.models.base import (
+    ImageClassificationHuggingFaceModel,
+)
+
+
+class SkinCancerHuggingFaceModel(ImageClassificationHuggingFaceModel):
+    """Wrapper class for Skin Cancer model on Hugging Face.
+
+    Args:
+        name (str): The name of the model.
+        device (str): The device to run the model on.
+    """
+
+    def __init__(self, name: str = None, device: str = "cpu"):
+        super().__init__(
+            model_id="Anwarkh1/Skin_Cancer-Image_Classification",
+            name=name,
+            device=device,
+        )
+
+    def predict_image(self, image) -> np.ndarray:
+        probas = super().predict_image(image)
```

Comment:
I would instead use `self.predict_probas` here

Reply:
Done in the single label base class